### ISSUE TITLE
Read Chef attributes within the tests ?

### ISSUE THREAD

**Issue by spuranam** [inspec/inspec#813]
### Description

[Issue 541](https://github.com/chef/inspec/issues/541) provided a way to consume chef attributes in the Inspec tests. But this workaround was broken by ChefDK 0.15.15.
### InSpec and Platform Version

ChefDK 0.15.15
OS: Windows 10
VirtualBox: 5.0.22
Vagrant: 1.8.4
### Replication Case

I have posted a reproduce able minimal project at https://github.com/spuranam/hello. 
### Stacktrace

```
-----> Starting Kitchen (v1.10.0)
-----> Verifying <default-leap>...
       Use `C:/Users/spuranam/chef-repo/cookbooks/hello/test/integration/default` for testing
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #verify action: [undefined method `[]' for nil:NilClass] on default-leap
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

**Comment by jeremymv2** [comment]
This can be accomplished with an inspec profile, which is how I've been achieving this for a while.

https://github.com/jeremymv2/test-inspec/blob/master/test/integration/profile/libraries/node.rb

It seems to be that `node = json('/tmp/kitchen_chef_node.json').params` when run in a non-inspec-profile is trying to open `/tmp/kitchen_chef_node.json` on the host machine, not in the TK instance and therefore you are getting the nil:NilClass error.

When using an inspec profile, this is not the case.

```
class NodeAttributes < Inspec::Resources::JsonConfig
  name 'node'
  def initialize
    super('/tmp/node.json')
  end
end
```

```
describe package('vim-minimal') do
  it { should be_installed }
  its('version') { should eq node.value(['vim','version']) }
end
```

**Comment by xTNTx** [comment]
@spuranam The core bug is described in issue above. It was present earlier but came up once default formatter was changed to `cli`. As temporary fix, change it back to `progress`, e.g. in your `.kitchen.yml`:

```
verifier:
  name: inspec
  format: progress
```

**Comment by xTNTx** [comment]
Fixed by #828

**Comment by chris-rock** [comment]
Please reopen, if the problem still occurs. Thanks for the additional testing @xTNTx
